datafusion/common/src/config.rs

```diff
@@ -303,21 +303,21 @@ config_namespace! {
         /// Valid values are: uncompressed, snappy, gzip(level),
         /// lzo, brotli(level), lz4, zstd(level), and lz4_raw.
         /// These values are not case sensitive.
```

Comment:
I think it would help to be clear what NULL means:

```suggestion
        /// These values are not case sensitive. If NULL, uses 
        /// default parquet writer setting
```